sgt-back-end/server.js

```diff
@@ -0,0 +1,197 @@
+import express from 'express';
+import pg from 'pg';
+
+// Create express app
+const app = express();
+
+// Use JSON middleware to parse request bodies
+app.use(express.json());
+
+// Create a pg database connection pool
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    // Allows non-SSL traffic to localhost
+    rejectUnauthorized: false,
+  },
+});
+
+// GET all rows from the grades table
+app.get('/api/grades', async (req, res, next) => {
+  try {
+    // Query the database
+    const sql = `
+    select *
+    from "grades";
+    `;
+
+    // Get results
+    const result = await db.query(sql);
+    const grades = result.rows;
+
+    // If no grades found, return a 404 error
+    if (!grades || grades.length === 0) {
```

Comment:
Typically in an API if there are no grades you just return an empty list.

Also, result.rows is guaranteed to be defined, so you can just return it without doing the `length === 0` check.